pages/index.vue

```diff
@@ -1,6 +1,11 @@
 <script lang="ts" setup>
-    if (!localStorage.getItem("auth-token")) {
-        navigateTo("/login");
+    if(localStorage.getItem('auth-token') === null){
+        if(localStorage.getItem('introduced') !== null){
```

Comment:
```suggestion
        if(localStorage.getItem('introduced') === "true"){
```
Ich weiß es ist nicht notwendig, aber es ist meiner meinung nach sehr viel leichter lesbar